display_list/display_list.cc

```diff
@@ -52,17 +52,105 @@ DisplayList::~DisplayList() {
   DisposeOps(ptr, ptr + byte_count_);
 }
 
+class Culler {
+ public:
+  virtual bool init(DispatchContext& context) = 0;
+  virtual void update(DispatchContext& context) = 0;
+};
+class NopCuller : public Culler {
+ public:
+  static NopCuller instance;
+
+  bool init(DispatchContext& context) override {
+    context.next_render_index = 0;
```

Comment:
nit: It would be helpful to potential maintainers in the future if a comment can be added here to explain why `next_render_index` is set to 0 here.

Reply:
Done. I added comments in a few places where DispatchContext is defined and modified.